neonvm/controllers/virtualmachine_controller.go

```diff
@@ -1401,3 +1513,44 @@ func getEnvVarValue(envVarName string) (string, error) {
 	}
 	return value, nil
 }
+
+func sshKeygen() ([]byte, []byte, error) {
```

Comment:
let's either add docs or name the return values or both? right now, I have to read the function body to know what the return values mean.

Reply:
fixed in fe99970e9c88fe1c4fa450c5503324e726da41ac